.gitignore

```diff
@@ -1,2 +1,5 @@
 node_modules
-.env
\ No newline at end of file
+.env
+dist
+calculator.js
+tasks.txt
```

Comment:
those 2 files shouldn't be there. any other developers wont have the same on theirs machines

Reply:
- [ ] delete useless lines